tasks/CentOS/main.yml

```diff
@@ -4,18 +4,18 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Set sensu_pkg_version {{ ansible_distribution }} override
```

Comment:
How does work? If sensu_pkg_version is none, won't this just set it to an empty string (the representation of nil in YAML)?